packages/plugin-core/src/commands/TaskComplete.ts

```diff
@@ -0,0 +1,44 @@
+import { ConfigUtils } from "@dendronhq/common-all";
+import { DENDRON_COMMANDS } from "../constants";
+import { BasicCommand } from "./base";
+import { VSCodeUtils, MessageSeverity } from "../vsCodeUtils";
+import { IDendronExtension } from "../dendronExtensionInterface";
+import { TaskStatusCommand } from "./TaskStatus";
+
+type CommandOpts = {};
+
+type CommandOutput = {} | undefined;
+
+export class TaskCompleteCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput
+> {
+  key = DENDRON_COMMANDS.TASK_COMPLETE.key;
+  public static requireActiveWorkspace: boolean = true;
+  private _ext: IDendronExtension;
+
+  constructor(extension: IDendronExtension) {
+    super();
+    this._ext = extension;
+  }
+
+  async execute(_opts: CommandOpts) {
+    const complete: string | undefined = ConfigUtils.getTask(
+      this._ext.getDWorkspace().config
+    ).taskCompleteStatus[0];
+
+    if (complete === undefined) {
+      await VSCodeUtils.showMessage(
+        MessageSeverity.ERROR,
+        "You have no task statuses marked as complete. Please add something to 'taskCompleteStatus' in your configuration file.",
```

Comment:
can we show a link to the relevant docs here and open dendron.yml?

Reply:
It actually looks like this never gets triggered, because the `ConfigUtils` fills in the configuration if it is empty or missing. I did add the link and config command for future proofing in case that changes in the future.